recipe/build.sh

```diff
@@ -1,17 +1,8 @@
 #!/bin/bash
 
 osname=`uname`
-if [ $osname == Linux ]; then
-    export CC="gcc"
-    export CXX="g++"
-elif [ $osname == Darwin ]; then
-    export CC="clang"
-    export CXX="clang++"
-fi
 
-export PKG_CONFIG_PATH="${PREFIX}/lib/pkgconfig/:"${PKG_CONFIG_PATH}
-export CFLAGS="-I${PREFIX}/include "${CFLAGS}
-export LDFLAGS="-L${PREFIX}/lib "${LDFLAGS}
+CFLAGS="-std=c11 ${CFLAGS} -Wno-implicit-function-declaration"
```

Comment:
Shouldn't  this be `CFLAGS="${CFLAGS} -std=c11 -Wno-implicit-function-declaration"`

Reply:
Or rather, if you want this to work: `export CFLAGS="${CFLAGS} -std=c11 -Wno-implicit-function-declaration"`